modules/login/php/requestaccount.class.inc

```diff
@@ -135,6 +135,13 @@ class RequestAccount extends \NDB_Form
             $errors['from'] = "Please provide a valid email!";
         } else if (!filter_var($_REQUEST['from'], FILTER_VALIDATE_EMAIL) ) {
             $errors['from'] = 'Please provide a valid email!';
+        } else if (preg_match('[<|>|"|&]', $_REQUEST['from'])) {
+            error_log("matched");
```

Comment:
Leftover debugging statement? If it's intentional it should be using a PSR logger instead.